<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F6F6F6;
border-radius : 20px solid,</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="repeat_day_radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 102, 102);
background-color: #F8E8EE;</string>
    </property>
    <property name="text">
     <string>Repeat daily</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #F8E8EE;
	border-radius : 20px solid,
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="create_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Create New Alarm</string>
     </property>
    </widget>
    <widget class="QLabel" name="alarm_time_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Alarm Time</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="alarm_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 20px solid;
background-color: #F6F6F6;
radius : 2px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="alarm_title_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Alarm Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="alarm_sound_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Alarm Sound</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:#F6F6F6;
	border-radius : 20px solid,
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="hour_comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="minuts_comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>33</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>34</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>35</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>36</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>37</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>39</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>41</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>42</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>43</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>44</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>45</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>46</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>47</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>49</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>51</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>52</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>53</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>54</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>55</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>58</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>59</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Clock_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/mahrayan/Downloads/clock-icon-transparent-free-icon-free-png.jpg</normaloff>C:/Users/mahrayan/Downloads/clock-icon-transparent-free-icon-free-png.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="song">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: #F8E8EE;
	border-radius : 20px,
}
</string>
     </property>
     <item>
      <property name="text">
       <string> jingle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> kirby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> tropical</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="play_unplay">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/mahrayan/Downloads/4249191.png</normaloff>C:/Users/mahrayan/Downloads/4249191.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #8785A2;
	color: rgb(255, 255, 255);
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_alarm_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #8785A2;
	color: rgb(255, 255, 255);
	border-radius: 20px ;
}</string>
     </property>
     <property name="text">
      <string>Set Alarm</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>repeat_day_radioButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
